software_reseller/__manifest__.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Software Reseller',
+    'version': 'saas~16.4.1.0',
+    'category': 'Studio',
+    'description': u"""
+This module has been generated by Odoo Studio.
+It contains the apps created with Studio and the customizations of existing apps.
+""",
+    'author': 'Odoo',
+    'depends': [
```

Comment:
I've looked at all data, and it seems that the real depends are those ones:

'project'
'knowledge'
'sale_purchase'
'sale_timesheet'
'sale_planning'
'sale_subscription'

Reply:
@fpodoo  In the demo database, CRM and Appointment are installed but do not have any data.
so, should we consider it or not?